### ISSUE TITLE
items taken from property

### ISSUE THREAD

**Issue by Truck624** [Uuvana-Studios/longvinter-windows-client#1392]
When I place items and it says only i can pick them up, people are able to steal them from my base even if they arent friends with me

**Comment by DemmyDemon** [comment]
Yes, people can still chainsaw your things to pick them up, it just takes them a while.
Turrets are the solution, though I'd love a server setting to fully protect camps.
